src/test/scala/com/google/cloud/spark/bigquery/BigQueryRelationProviderSuite.scala

```diff
@@ -84,4 +89,3 @@ class BigQueryRelationProviderSuite extends FunSuite with BeforeAndAfter with Mo
     verify(bigQuery).getTable(Matchers.eq(ID))
   }
 }
-
```

Comment:
Add a test case covering `credentials` option.

Reply:
@cyxxy - can you take a look into this? If you would like to merge this sooner, at least create an issue and leave a todo here.

Reply:
Created an issue and added a to-do.